test/unit/feed_diff.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
Consider moving these sample data files to a fixtures directory to avoid clutter as this repo grows in size.